src/main/webapp/app/exercises/shared/result/result-detail.component.ts

```diff
@@ -341,7 +342,7 @@ export class ResultDetailComponent implements OnInit {
     private createProgrammingExerciseScaFeedbackItem(feedback: Feedback): FeedbackItem {
         const scaCategory = feedback.text!.substring(STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER.length);
         const scaIssue = StaticCodeAnalysisIssue.fromFeedback(feedback);
-        const text = this.showTestDetails ? `${scaIssue.rule}: ${scaIssue.message}` : scaIssue.message;
+        const text = `${scaIssue.rule}: ${scaIssue.message}`;
```

Comment:
Was this change on purpose? If yes, why?

Reply:
This allows the students to actually see which SCA feedback belongs to which category. Since this PR now actually shows the amount of deducted points correctly if the student reaches the deduction limit, this information is necessary to see what other issues belong to the same category.
Beside that, this is no hidden information. Every student could deduct the SCA categories from the feedback if they want to.

Reply:
The category is not the rule here. You can see the difference (tutor view vs student view) below. 

![grafik](https://user-images.githubusercontent.com/26540346/172710116-3c368678-ecf1-43c5-b925-aa85a50f42c8.png)
![grafik](https://user-images.githubusercontent.com/26540346/172710152-4c284fa3-2b9e-474d-9634-ea2696014ba9.png)

I also think that some names could get confusing for the students: 
![grafik](https://user-images.githubusercontent.com/26540346/172713936-45560967-76db-40b6-bb18-aeba1830b0fa.png)

Reply:
You're right!